src/agent/src/command_handler_utils.hpp

```diff
@@ -34,7 +31,7 @@ DispatchCommand(module_command::CommandEntry commandEntry,
 /// @param module The module that will execute the command
 /// @param messageQueue The message queue to send the result to
 /// @return The result of the command execution
-boost::asio::awaitable<module_command::CommandExecutionResult>
+boost::asio::awaitable<std::optional<module_command::CommandExecutionResult>>
 DispatchCommand(module_command::CommandEntry commandEntry,
                 std::shared_ptr<ModuleWrapper> module,
                 std::shared_ptr<IMultiTypeQueue> messageQueue);
```

Comment:
You can remove this parameter if previous comment applies.

Reply:
Removed!